.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,4 @@
+contact_links:
+  - name: "Getting Help"
```

Comment:
```suggestion
  - name: "Get help"
```

Reply:
There is currently a general push towards more direct/active language, particularly around links etc